SolrNet/Impl/AbstractSolrQueryResults.cs

```diff
@@ -56,6 +56,11 @@ public abstract class AbstractSolrQueryResults<T> : List<T> {
         /// </summary>
         public IDictionary<string, ICollection<KeyValuePair<string, int>>> FacetIntervals { get; set; }
 
+        /// <summary>
+        /// Facet aggregation functions results
+        /// </summary>
+        public dynamic FacetFunctions { get; set; }

```

Comment:
No `dynamic` please. Let's use a type as precise as possible.

Reply:
My initial implementation was different but then I figured that response can be really dynamic. I can try to avoid it but I believe we will end up with `IDictionary<string, object>` at best.
Need more time for it - to test all the features/cases.